packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -469,8 +438,9 @@ class SemanticsNode extends AbstractNode {
     buffer.write('; $rect');
     if (wasAffectedByClip)
       buffer.write(' (clipped)');
-    for (SemanticsAction action in _actions) {
-      buffer.write('; $action');
+    for (SemanticsAction action in SemanticsAction.values.values) {
```

Comment:
Yeah, I was originally trying to make `SemanticsAction` look as much like an enum as possible, but it's not really what you want because `values` is a map...